File: king-admin-java/src/main/java/com/oukingtim/domain/TbCalendar.java
package com.oukingtim.domain;

import com.baomidou.mybatisplus.annotations.TableName;
import lombok.Data;
import lombok.EqualsAndHashCode;

/**
 * Created by oukingtim
 */
@TableName("tb_calendar")
@Data
@EqualsAndHashCode(callSuper = false)
public class TbCalendar extends BaseModel<TbCalendar> {

    private String title;
    private String start;
    private String end;
    private String color;

}

File: king-admin-kotlin/src/main/kotlin/com/oukingtim/domain/TbCalendar.kt
package com.oukingtim.domain

import com.baomidou.mybatisplus.annotations.TableName

/**
 * Created by oukingtim
 */
@TableName("tb_calendar")
class TbCalendar : BaseModel<TbCalendar>() {
    val title: String? = null
    val start: String? = null
    val end: String? = null
    val color: String? = null

}